include/llvm/Transforms/Tapir/TapirTypes.h

```diff
@@ -21,9 +21,10 @@ enum class TapirTargetType {
   None = 0,
   Serial = 1,
   Cilk = 2,
-  OpenMP = 3,
-  CilkR = 4,
-  Qthreads = 5
+  CilkLegacy = 3,
```

Comment:
Are we sure we want new targets for the same backends with different loop strategies?

Reply:
I'm leaning towards doing so, with cilk legacy denoting the ABI version of the divide and conquer and Cilk denoting our version (building in the compiler). Therefore it you could have any of the 4 (serial vs DAC) x (Cilk vs CilkLegacy), each with different meanings. There are a couple of small differences between the Cilk/Cilk Legacy (such as passing in everything via registers vs a struct and loading), and if the code is there it would be nice to have an explicit path to enable it for an apples-to-apples comparison w other compilers that simply use the abi.